content/exchange/artifacts/Exchange.Windows.System.PowerShell.DetectResponder

```diff
@@ -0,0 +1,34 @@
+name: Custom.Windows.System.PowerShell.DetectResponder
+author: "Dhruv Majumdar, Jamie Bhoohe"
+description: |
+  This artifact allows to detect responder in the environment
+type: CLIENT_EVENT
+required_permissions:
+  - EXECVE
+
+precondition:
+  SELECT OS From info() where OS = 'windows'
+
+parameters:
+  - name: Command
+    default: "dir C:/"
+  - name: PowerShellExe
+    default: "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
+  - name: ReloadPeriod
+    description: Checks for responder activity
+    default: "600"
+    type: int
+
+sources:
+  - query: |
+      SELECT * FROM foreach(
+        row={
+           SELECT * FROM clock(period=ReloadPeriod)
+        },
+        query={
+          SELECT * FROM execve(argv=[PowerShellExe,
+        "-ExecutionPolicy", "Unrestricted", "-encodedCommand",
+        "JABsAGwAbQBuAHIAIAA9ACAAKABSAGUAcwBvAGwAdgBlAC0ARABuAHMATgBhAG0AZQAgAC0ATABsAG0AbgByAE8AbgBsAHkAIABlAHYAaQBsAEQAZQB4AHQAZQByACkAIAAyAD4AIAAkAE4AVQBMAEwACgBpAGYAIAAoACQAbABsAG0AbgByACkAIAAgAHsACgAgACAAIAAgACQAZQB2AGkAbABfAEkAUAAgAD0AIAAkAGwAbABtAG4AcgAuAEkAUABBAGQAZAByAGUAcwBzACAALQBKAG8AaQBuACAAIgAsACAAIgAKACAAIAAgACAAJABtAHMAZwAgAD0AIAAiAFMAdQBiAGoAZQBjAHQAOgAgAEgASQBHAEgAIABTAEUAVgAgAC0AIABSAGUAcwBwAG8AbgBkAGUAcgAgAEQAZQB0AGMAdABlAGQAIABgAG4ARABvAG0AYQBpAG4AOgAgACQAZQBuAHYAOgBVAFMARQBSAEQATgBTAEQATwBNAEEASQBOACAAYABuAEgAbwBzAHQAbgBhAG0AZQA6ACAAJAB7AGUAbgB2ADoAYwBvAG0AcAB1AHQAZQByAG4AYQBtAGUAfQAgAGAAbgBSAG8AdQBnAGUAIABMAEwATQBOAFIAIABTAGUAcgB2AGUAcgA6ACAAJABlAHYAaQBsAF8ASQBQACIACgBlAGMAaABvACAAJABtAHMAZwAKAH0A"])
```

Comment:
Please do not include opaque blobs in artifacts in the exchange. It is hard to see what its doing

Reply:
$llmnr = (Resolve-DnsName -LlmnrOnly evilDexter) 2> $NULL
if ($llmnr)  {
    $evil_IP = $llmnr.IPAddress -Join ", "
    $msg = "Subject: HIGH SEV - Responder Detcted `nDomain: $env:USERDNSDOMAIN `nHostname: ${env:computername} `nRouge LLMNR Server: $evil_IP"
	echo $msg
}

//The unobfuscated code.